linkerd/app/inbound/src/direct.rs

```diff
@@ -70,6 +70,8 @@ where
     let http = svc::stack(http_loopback)
         .push_on_response(
             svc::layers()
+                .push(svc::layer::mk(svc::SpawnReady::new))
```

Comment:
could adding a `SpawnReady` here change behavior meaningfully?

Reply:
Not really. It's just slightly more robust, as we have a buffer/failfast here already